scripts/sns/install/upgrade_dapp.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+# This script is used to create an SNS proposal for increasing the cost of proposal rejection to 200 MOD
+# This script is called by upgrade_test_canister.sh
+# For detailed explanation please see the GitHub Repo: https://github.com/dfinity/sns-testing/tree/main
+
+
+set -euo pipefail
+
+CURRENTDIR="$(pwd)"
+
+cd -- "$(dirname -- "${BASH_SOURCE[0]}")"
+
+REPODIR="$(pwd)"
```

Comment:
overall the pr lgtm.
The variable name `REPODIR` is confusing though.
I thought `cd -- "$(dirname -- "${BASH_SOURCE[0]}")"`would redirect you to the same folder of the script.
I suggest to use SCRIPT_DIR